### DISCUSSION TITLE
List of URLs of MRFs

### DISCUSSION THREAD

**Discussion by cheernurse** [CMSgov/price-transparency-guide#566] [General]
I have spent a lot of time searching for the URLs of health plan's and hospital's MRFs. 

Anyone else out there doing the same thing? 
Can we share a Google Sheet?

-wen 480-815-1628   cheernurse@gmail.com

**Comment by AshleyOboe** [comment]
Hi @cheernurse - I'm looking for this as well, did you find any source?  If not I can get an analyst to help put one together but maybe we can coordinate.  Thanks!

**Comment by cheernurse** [comment]
Hi, Ashley,

I haven't. I'm in Switzerland on vacation and will be back to San Francisco
this Sunday.

Can we schedule a call for next week?

480-815-1628

**Comment by dgolden** [comment]
Hi Wen,
Please include me in the discussion. 
Thanks, 
David dgolden@keyark.com

**Reply by AshleyOboe** [reply]
Hey guys sorry I meant to schedule something but for now I've just been working on BUCA, so found all of those.  I'll let you know if I come across a more complete list when I'm done.

**Comment by kulnor** [comment]
Just to let you know we recently started a GutHub repo around this issue:
https://github.com/postman-open-technologies/us-cms-price-transparency

At this time simply compiling a list of MRFs sources in the readme file, but looking into various options to compile this. If you have more links, don't hesitate to update directly and simply send my way and I'll add. Will keep you posted on progress.

**Reply by alecstein** [reply]
I've started this Google sheet and seeded it with some basic information. You all should feel free to add to /modify it.
https://docs.google.com/spreadsheets/d/1Uuphuk41tFGmrulXhfeDN2zVkmVLzb4S1kdq20-4uZo/edit#gid=0

**Reply by riashah28** [reply]
This is great - thanks for sharing!

**Reply by harrymcneill** [reply]
this is great - thank you.
ive noticed some insurers have one (or two files) available per in/out of network, and others such as United have hundreds....i think they have it by provider maybe.
is this how you see it also?
